tests/cypress/support/commands.js

```diff
@@ -0,0 +1 @@
+require("cy-verify-downloads").addCustomCommand();
```

Comment:
For one line, might as well just throw it in `tests/cypress/support/index.js` with the others. 🤷